delocate/delocating.py

```diff
@@ -235,18 +282,24 @@ def _copy_required(lib_path, copy_filt_func, copied_libs):
 
 
 def _dylibs_only(filename):
+    # type: (Text) -> bool
     return (filename.endswith('.so') or
             filename.endswith('.dylib'))
 
 
 def filter_system_libs(libname):
+    # type: (Text) -> bool
     return not (libname.startswith('/usr/lib') or
                 libname.startswith('/System'))
 
 
-def delocate_path(tree_path, lib_path,
-                  lib_filt_func=None,
-                  copy_filt_func=filter_system_libs):
+def delocate_path(
+    tree_path,  # type: Text
+    lib_path,  # type: Text
+    lib_filt_func=None,  # type: Union[None, str, Callable[[Text], bool]]
```

Comment:
Admittedly longer, but maybe more explicit as:

```suggestion
    lib_filt_func=None,  # type: Optional[Union[str, Callable[[Text], bool]]]
```